common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -1,241 +1,71 @@
+        if (!(this instanceof VideoTranscriptDownloadHandler)) {
+            return new VideoTranscriptDownloadHandler(element, options);
         }
 
-        _.bindAll(this, 'openMenu', 'openMenuHandler', 'closeMenu', 'closeMenuHandler', 'toggleMenuHandler',
-            'clickHandler', 'keyDownHandler', 'render', 'menuItemsLinksFocused', 'changeFileType', 'setValue'
-        );
+        _.bindAll(this, 'bindHandlers', 'clickHandler');
 
         this.container = element;
         this.options = options || {};
 
-        if (this.container.find('.video-tracks')) {
+        if (this.container.find('.wrapper-downloads .wrapper-download-transcripts')) {
             this.initialize();
         }
+
+        return false;
     };
 
-    VideoAccessibleMenu.prototype = {
-        /** Initializes the module. */
+    VideoTranscriptDownloadHandler.prototype = {
+        // Initializes the module.
         initialize: function() {
             this.value = this.options.storage.getItem('transcript_download_format');
-            this.el = this.container.find('.video-tracks .a11y-menu-container');
-            this.render();
+            this.el = this.container.find('.list-download-transcripts');
             this.bindHandlers();
         },
 
-        /**
-         * Creates any necessary DOM elements, attach them, and set their,
-         * initial configuration.
-         */
-        render: function() {
-            var value, msg;
-            // For the  time being, we assume that the menu structure is present in
-            // the template HTML. In the future accessible menu plugin, everything
-            // inside <div class='menu-container'></div> will be generated in this
-            // file.
-            this.button = this.el.children('.a11y-menu-button');
-            this.menuList = this.el.children('.a11y-menu-list');
-            this.menuItems = this.menuList.children('.a11y-menu-item');
-            this.menuItemsLinks = this.menuItems.children('.a11y-menu-item-link');
-            value = (function(val, activeElement) {
-                return val || activeElement.find('a').data('value') || 'srt';
-            }(this.value, this.menuItems.filter('.active')));
-            msg = '.' + value;
-
-            if (value) {
-                this.setValue(value);
-                this.button.text(gettext(msg));
-            }
-        },
-
-        /** Bind any necessary function callbacks to DOM events. */
+        // Bind any necessary function callbacks to DOM events.
         bindHandlers: function() {
-            // Attach various events handlers to menu container.
-            this.el.on({
-                'mouseenter': this.openMenuHandler,
-                'mouseleave': this.closeMenuHandler,
-                'click': this.toggleMenuHandler,
-                'keydown': this.keyDownHandler
-            });
-
             // Attach click and keydown event handlers to individual menu items.
```

Comment:
This comment no longer makes sense. As I mentioned above, let's ditch the separate "bindHandlers" method an inline the single line of code.